audit/projects/kubernetes-public/services/compute/project-info.json

```diff
@@ -1,22 +1,22 @@
 {
   "commonInstanceMetadata": {
-    "fingerprint": "9_em4FUS_lU=",
```

Comment:
Another candidate for being removed  @bartsmykla

Reply:
ack

Reply:
Done: #809